services/horizon/internal/flags.go

```diff
@@ -391,8 +391,15 @@ func Flags() (*Config, support.ConfigOptions) {
 			Name:        "ingest",
 			ConfigKey:   &config.Ingest,
 			OptType:     types.Bool,
-			FlagDefault: true,
-			Usage:       "causes this horizon process to ingest data from stellar-core into horizon's db",
+			FlagDefault: false,
```

Comment:
Maybe we should add Action Needed tag somewhere here:
```suggestion
			// Action needed in release: horizon-v2.9.0: make --ingest default true
			FlagDefault: false,
```